app/assets/javascripts/to_do_items.coffee

```diff
@@ -40,14 +41,56 @@ class @TodoItem
   detectCancelFormButton = (id) ->
     $("#cancel-item-button-#{id}")
 
+  extractId = (str) ->
+    str.split('-').pop()
+
   removeCallout = (listId) ->
     detectList(listId).find('div.bs-callout').remove()
 
+  saveOnClick = (item) ->
+    $(item).click () ->
+      updateItem extractId(item['id']), item['value']
```

Comment:
@vlasiak, I suggest to use `check` (or `toggle`) term instead of `save` and `update`.